File: Modul 3/C/PRAK305-2310817310010-AkmallullailSyaban.c
#include <stdio.h>

int main (){
    printf("input\n");
    int CekHari, CekJam, CekMenit, detik;
    scanf("%d",&detik);
    CekHari = detik / 86400;
    detik = detik % 86400;
    CekJam = detik / 3600;
    detik = detik % 3600;
    CekMenit = detik / 60;
    detik = detik % 60;
    if (CekHari > 0){
        printf("Output\n");
        printf("%d hari %.2d:%.2d:%.2d\n",CekHari,CekJam,CekMenit,detik);
    }
    else {
        printf("Output\n");
        printf("%.2d:%.2d:%.2d\n",CekJam,CekMenit,detik);
    }
}

File: Modul 3/Python/PRAK305-2310817310010-AkmallullailSyaban.py
print("Input")
x = int(input())

CekHari = x // 86400
detik = x % 86400

CekJam = detik // 3600
detik = detik % 3600

CekMenit = detik // 60
detik = detik % 60

print("Output")
if CekHari > 0:
    print(f"{CekHari} hari {CekJam:02}:{CekMenit:02}:{detik:02}")
else:
    print(f"{CekJam:02}:{CekMenit:02}:{detik:02}")
